hw/ip/hmac/doc/checklist.md

```diff
@@ -98,17 +98,17 @@ Security      | [SEC_CM_COUNCIL_REVIEWED][]  | N/A         | This block only con
 
  Type         | Item                    | Resolution  | Note/Collaterals
 --------------|-------------------------|-------------|------------------
-Documentation | [NEW_FEATURES_D3][]     | Done        |
-RTL           | [TODO_COMPLETE][]       | Done        |
-Code Quality  | [LINT_COMPLETE][]       | Done        |
-Code Quality  | [CDC_COMPLETE][]        | N/A         |
-Code Quality  | [RDC_COMPLETE][]        | N/A         |
-Review        | [REVIEW_RTL][]          | Done        |
-Review        | [REVIEW_DELETED_FF][]   | N/A         |
-Review        | [REVIEW_SW_CHANGE][]    | N/A         |
-Review        | [REVIEW_SW_ERRATA][]    | N/A         |
-Review        | Reviewer(s)             | Done        | @rswarbrick @cindychip @jadephilipoom
-Review        | Signoff date            | Done        | 2022-07-27
+Documentation | [NEW_FEATURES_D3][]     |             |
```

Comment:
The default value for things not done yet is `Not Started`. Can you please update the PR accordingly?